src/app/docs/colorpicker/colorpicker-docs.component.html

```diff
@@ -8,14 +8,38 @@
   </sky-docs-demo-page-summary>
 
   <sky-docs-demo>
-    See documentation for how to setup demos.
+    <sky-colorpicker
+      #colorPicker
+    >
+      <input
+        name="favoriteColor"
+        type="text"
+        [skyColorpickerInput]="colorPicker"
+        [(ngModel)]="model.favoriteColor"
+      />
+    </sky-colorpicker>
   </sky-docs-demo>
 
   <sky-docs-design-guidelines>
     See documentation for how to setup design guidelines.
   </sky-docs-design-guidelines>
 
-  <sky-docs-code-examples>
-    See documentation for how to setup code examples.
+  <sky-docs-code-examples
+    [packageDependencies]="{
+      '@skyux/colorpicker': '*'
+    }"
+  >
+    <sky-docs-code-example
+      heading="Reactive colorpicker"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/colorpicker/reactive"
+    ></sky-docs-code-example>
+    <sky-docs-code-example
+      heading="Template-driven colorpicker"
```

Comment:
This was a discussion I had with Todd/Adam several months ago, but we've decided to remove the template-driven forms entirely and only demo the reactive forms.

Reply:
Noted. I've removed this.